src/Aspire.Hosting/DistributedApplicationBuilder.cs

```diff
@@ -76,6 +76,7 @@ public DistributedApplicationBuilder(DistributedApplicationOptions options)
 
         _innerBuilder.Logging.AddFilter("Microsoft.Hosting.Lifetime", LogLevel.Warning);
         _innerBuilder.Logging.AddFilter("Microsoft.AspNetCore.Server.Kestrel", LogLevel.Error);
+        _innerBuilder.Logging.AddFilter("Aspire.Hosting.Dashboard", LogLevel.Error);
```

Comment:
```suggestion
        _innerBuilder.Logging.AddFilter("Aspire.Hosting.Dashboard", LogLevel.Warning);
```
I think we should be showing warnings as well. If we don't show them, what value would they be? No one would ever see them.

Note that Kestrel being filtered to only Error is really a bug that we should fix.

Reply:
Should we do the config thing or do we just want to hardcode it?

Reply:
IMO hard-coding it is fine. Especially becuase 90%+ of the time people don't want to see the dashboard logs.

Is it possible for a config setting to override the hard-coded value? That would be ideal.

Reply:
I thought that was the case but I couldn't get it to override.

Reply:
Done